moped-editor/src/components/GridTable/FiltersChips.js

```diff
@@ -101,6 +118,7 @@ const FiltersChips = ({
             <Grid item>
               <Chip
```

Comment:
@chiaberry could you also make the color="primary" on this isOr chip? John had brought it up in [the discussion](https://austininnovation.slack.com/archives/CNUEPKLB1/p1718826221462969) a while back, and I think it would look great now that I'm seeing this live.